exercises/04_count_rune/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+func CountRune(s string, b rune) int {
+	var count int
+	s1 := []rune(s)
```

Comment:
nit:

1) You don't need any temporary variables for this
```go
for _, c := range []rune(s) { ... }
```

2) You don't even have to cast, since `for-range` defaults to `rune`:
```go
for _, c := range s {
  if c == b { count++ }
}
```